cylc/flow/suite_status.py

```diff
@@ -65,6 +65,9 @@ class StopMode(Enum):
     REQUEST_CLEAN = 'REQUEST(CLEAN)'
     """External shutdown request, will wait for active jobs to complete."""
 
+    REQUEST_KILL = 'REQUEST(KILL)'
+    """External shutdown request, will wait for active jobs to be killed."""
```

Comment:
Not the fault of this PR (`git blame` fingers @oliver-sanders) but I don't think we doing this in other modules (are we?) and in fact last I heard Python doesn't have attribute docstrings - the PEP was rejected years ago.  So these strings should probably just be comments before the attributes?  (Or is this for Sphinx doc generation? I'm sure @oliver-sanders will enlighten me!)

Reply:
Yeah, it appears only the describe method is used to get different strings (?):
https://github.com/dwsutherland/cylc-flow/blob/e4c6e4cc9571893d6c7b9523d0cb1dbec9c383cc/cylc/flow/network/schema.py#L1297-L1308

Reply:
These strings do nothing in Python, however, Sphinx has some special parsing logic which picks them up which makes for a convenient way of documenting them.

Reply:
Should we have an issue up to extend this throughout the code then?